.php_cs

```diff
@@ -38,46 +38,47 @@ $config->setRules(
     [
         '@PSR2' => true,
         'align_multiline_comment' => ['comment_type' => 'all_multiline'],
+        'array_indentation' => true,
         'array_syntax' => ['syntax' => 'short'],
         'blank_line_before_statement' => [
             'statements' => ['return', 'continue', 'break', 'declare', 'throw', 'try'],
         ],
-        'cast_spaces' => ['space' => 'none'],
-        'combine_consecutive_issets' => true,
-        'combine_consecutive_unsets' => true,
+        'binary_operator_spaces' => true,
+        'cast_spaces' => ['space' => 'single'],
```

Comment:
👎

Reply:
`'single'` is a default and also preferred in [PHP documentation](http://www.php.net/manual/en/language.types.type-juggling.php#language.types.typecasting).

Reply:
It may be so - however, I tend to go for defaults that protect a user when copying/pasting or refactoring manually.

It'd be quite easy to take just the variable and not the cast when the variable is separated from the cast - hence keeping them together.

Reply:
While it's not specified in the [extended coding style proposal](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#6-operators), given that `cast` is a binary operator, it should be followed by a space for consistency.

Also, I don't think copy-paste argument makes sense, because if someone is copying a variable, why would they need to take type casting along? Or, if they did, then casting has a purpose in some wider context, and that would mean more code to the left from cast operator.

Reply:
Cast isn't a binary operator, it's a unary operator.  it only requires one parameter.

Therefore, it should be treated the same as other unary operators - so we should be doing

```
$i ++;
++ $i;
$x = - 5;
```

if we go by that ruling

Reply:
My mistake, the cast operator is unary. Therefore proposal doesn't cover it.

I'm going to change it so we can move on.

Reply:
See a51899c7fd8643c735bf5b35dd7b5686949acea7